src/main/java/com/google/alpollo/AnalysisHelper.java

```diff
@@ -1,8 +1,16 @@
 package com.google.alpollo;
 
```

Comment:
Now I see that all methods in this class are `static`, so it's good to make it stateless

This's called Utility class and you can find a lot of best practises for it, for example, [here](https://stackoverflow.com/questions/31409982/java-best-practice-class-with-only-static-methods)